hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/ThreadLocalContext.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.hadoop.hdfs.server.federation.router;
```

Comment:
@KeeProMise Sir, we should add license header here.

Reply:
@hfutatzhanghb Thank you for your careful review, i will add this later!